### ISSUE TITLE
CT: Assertion failed: If this is the first alert, then we have problems

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/friction#188]
Intermittent failure in CT, looks like it's related to an a11y Describer.

```
friction : phet-io-studio-fuzz : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1601362030461/studio/?sim=friction&phetioDebug&fuzz&wrapperContinuousTest=%7B%22test%22%3A%5B%22friction%22%2C%22phet-io-studio-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1601362030461%22%2C%22timestamp%22%3A1601383531892%7D
Uncaught Error: Uncaught Error: Assertion failed: If this is the first alert, then we have problems
Error: Assertion failed: If this is the first alert, then we have problems
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1601362030461/assert/js/assert.js:22:13)
at BreakAwayDescriber.alertAtEvaporationThreshold (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1601362030461/friction/js/friction/view/describers/BreakAwayDescriber.js:72:17)
at BreakAwayDescriber.amplitudeListener (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1601362030461/friction/js/friction/view/describers/BreakAwayDescriber.js:55:14)
at TinyEmitter.emit (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1601362030461/axon/js/TinyEmitter.js:71:18)
at NumberProperty._notifyListeners (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1601362030461/axon/js/Property.js:277:25)
at NumberProperty.set (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1601362030461/axon/js/Property.js:175:14)
at https://bayes.colorado.edu/continuous-testing/ct-snapshots/1601362030461/friction/js/friction/model/FrictionModel.js:269:39
at TinyEmitter.emit (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1601362030461/axon/js/TinyEmitter.js:71:18)
at Vector2Property._notifyListeners (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1601362030461/axon/js/Property.js:277:25)
at Vector2Property.set (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1601362030461/axon/js/Property.js:175:14)
id: Bayes Chrome
Snapshot from 9/29/2020, 12:47:10 AM
```

**Comment by zepumph** [comment]
@jessegreenberg, I see commits in GrabDragInteraction from https://github.com/phetsims/scenery-phet/issues/622, do you think this studio-fuzz problem is due to that interrupt work?

**Comment by KatieWoe** [comment]
I'm seeing this 11/6/2020:
```
riction : phet-io-studio-fuzz : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1604652787156/studio/?sim=friction&phetioDebug&fuzz&wrapperContinuousTest=%7B%22test%22%3A%5B%22friction%22%2C%22phet-io-studio-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1604652787156%22%2C%22timestamp%22%3A1604671577067%7D
Uncaught Error: Uncaught Error: Assertion failed: If this is the first alert, then we have problems
Error: Assertion failed: If this is the first alert, then we have problems
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1604652787156/assert/js/assert.js:22:13)
at BreakAwayDescriber.alertAtEvaporationThreshold (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1604652787156/friction/js/friction/view/describers/BreakAwayDescriber.js:72:17)
at BreakAwayDescriber.amplitudeListener (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1604652787156/friction/js/friction/view/describers/BreakAwayDescriber.js:55:14)
at TinyEmitter.emit (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1604652787156/axon/js/TinyEmitter.js:71:18)
at NumberProperty._notifyListeners (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1604652787156/axon/js/Property.js:280:25)
at NumberProperty.set (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1604652787156/axon/js/Property.js:178:14)
at https://bayes.colorado.edu/continuous-testing/ct-snapshots/1604652787156/friction/js/friction/model/FrictionModel.js:268:41
at TinyEmitter.emit (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1604652787156/axon/js/TinyEmitter.js:71:18)
at Vector2Property._notifyListeners (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1604652787156/axon/js/Property.js:280:25)
at Vector2Property.set (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1604652787156/axon/js/Property.js:178:14)
id: Bayes Chrome
Snapshot from 11/6/2020, 1:53:07 AM
```

**Comment by zepumph** [comment]
I haven't seen this on CT in ages, and after 10 minutes of studio fuzzing locally there is no error. I'm going to close, but will reopen if it comes up again.

**Comment by KatieWoe** [comment]
This seems to have happened again:
```
friction : phet-io-state-fuzz : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1627831135671/phet-io-wrappers/state/?sim=friction&phetioDebug&fuzz&wrapperContinuousTest=%7B%22test%22%3A%5B%22friction%22%2C%22phet-io-state-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1627831135671%22%2C%22timestamp%22%3A1627909650536%7D
Uncaught Error: Uncaught Error: Assertion failed: If this is the first alert, then we have problems
Error: Assertion failed: If this is the first alert, then we have problems
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1627831135671/assert/js/assert.js:25:13)
at BreakAwayDescriber.alertAtEvaporationThreshold (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1627831135671/friction/js/friction/view/describers/BreakAwayDescriber.js:74:17)
at BreakAwayDescriber.amplitudeListener (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1627831135671/friction/js/friction/view/describers/BreakAwayDescriber.js:56:14)
at TinyProperty.emit (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1627831135671/axon/js/TinyEmitter.js:86:9)
at NumberProperty._notifyListeners (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1627831135671/axon/js/Property.js:271:23)
at NumberProperty.set (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1627831135671/axon/js/Property.js:186:14)
at https://bayes.colorado.edu/continuous-testing/ct-snapshots/1627831135671/friction/js/friction/model/FrictionModel.js:275:41
at TinyProperty.emit (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1627831135671/axon/js/TinyEmitter.js:86:9)
at Vector2Property._notifyListeners (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1627831135671/axon/js/Property.js:271:23)
at Vector2Property.set (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1627831135671/axon/js/Property.js:186:14)
id: Bayes Chrome
Snapshot from 8/1/2021, 9:18:55 AM
```

**Comment by zepumph** [comment]
Thanks!

**Comment by zepumph** [comment]
Instead of having this be an assertion, I made this case be a guard, so that we can have the more informative alerts occur first before the total breakaway alert. It is quite easy to break away all atoms for one alert.

I removed the assertion, we won't see it on CT anymore.
